### PR TITLE
only to tribo_tech review

### PR DESCRIPTION
**Pull request by fbtt**

### CODE REVIEW

File: my_app/urls.py [hunk 1]
```diff
@@ -3,4 +3,5 @@
 
 urlpatterns = [
     path('', views.home, name='home'),
-]
\ No newline at end of file
+    path('new_check', views.new_check, name='new_check'),
```
Comment (lfpelison) [inline my_app/urls.py:6]: is it new_check or news_check?

File: my_app/views.py [hunk 2]
```diff
@@ -4,4 +4,18 @@
...
+
+
+def new_check(request):
+
+    news_content = request.POST.get('news_content')
+
+    # Consultar modelo e retornar veracidade
+    veracidade = 'FALSA'
+
+    stuff_for_frontend = {
```
Comment (igoryamamoto) [inline my_app/views.py:17]: little bit informal this variable name, right? 😂

Maybe `frontend_context` would be a good fit to represent what this variable does

File: my_app/views.py [hunk 3]
```diff
@@ -4,4 +4,18 @@
...
\ No newline at end of file
+    return render(request, template_name='base.html')
+
+
+def new_check(request):
+
+    news_content = request.POST.get('news_content')
+
+    # Consultar modelo e retornar veracidade
+    veracidade = 'FALSA'
```
Comment (lfpelison) [inline my_app/views.py:15]: veracidade is in portuguese and the code is in english

File: static/css/style.css [hunk 4]
```diff
@@ -0,0 +1,24 @@
+.card {
+    height: 400px; /* Your height here */
```
Comment (lfpelison) [inline static/css/style.css:2]: maybe these comments are not necessary

File: static/css/style.css [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    overflow: hidden;
+}
+
+.card-image {
+    height: 250px; /* Your height here */
```
Comment (lfpelison) [inline static/css/style.css:7]: same here

File: my_app/views.py (hunk 3 above)
Comment (fbtt) [reply my_app/views.py:15]: ok

File: static/css/style.css (hunk 4 above)
Comment (fbtt) [reply static/css/style.css:2]: ok

File: static/css/style.css (hunk 5 above)
Comment (fbtt) [reply static/css/style.css:7]: ok

**Pull request merged**